app/assets/config/decidim_challenges_manifest.js

```diff
@@ -1,3 +1,5 @@
 //= link decidim/challenges/add_challenge.js
 //= link decidim/challenges/admin/challenges_form.js
-//= link_directory ../stylesheets/decidim/challenges .css
\ No newline at end of file
+//= link_directory ../stylesheets/decidim/challenges .css
```

Comment:
Is this space between `challenges` and `.css` left on purpose?
Same with `solutions` and `.css`